code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -410,22 +410,20 @@
 		return
 
 	var/embeds = FALSE
-	for(var/X in bodyparts)
-		var/obj/item/bodypart/LB = X
-		for(var/obj/item/I in LB.embedded_objects)
+	for(var/obj/item/bodypart/limb as anything in bodyparts)
+		for(var/obj/item/weapon in limb.embedded_objects)
```

Comment:
Can we `as anything` `embedded_objects` or do we hold projectiles in there or something